framework/src/modules/http_api/controllers/node.js

```diff
@@ -71,6 +71,24 @@ function _filterTransactions(transactions, filters) {
 				return left.type - right.type;
 			});
 		}
+		if (key === 'nonce') {
+			filteredTransactions.sort((a, b) => {
+				let left = a;
+				let right = b;
+				if (order === 'desc') {
+					left = b;
+					right = a;
+				}
+				const diff = BigInt(left.nonce) - BigInt(right.nonce);
+				if (diff > BigInt(0)) {
+					return 1;
+				}
+				if (diff < BigInt(0)) {
+					return -1;
+				}
+				return 0;
+			});
+		}
```

Comment:
Should not we make this sorting logic more generic so we can reuse it. Here its the exactly same logic compared to sorting with `fee`.

Reply:
true. but this is already removed logic in v5. I think it doesn't have so much benefit in making it generic?